app/controllers/disability_claims_base_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class DisabilityClaimsBaseController < ApplicationController
+  before_action :authorize_user
+
+  protected
+
+  def authorize_user
+    head(403) unless @current_user.evss_attrs?
```

Comment:
Note @austin-martinez I added an exception for Forbidden to Common::Exceptions::Forbidden that you can use once my PR gets merged up.